EnglishFilter/sections/antiadblock.txt

```diff
@@ -291,7 +291,7 @@ cmacapps.com##cloudflare-app[app-id="no-adblock"]
 ! JS for the sites with frequenly changed ad domains
 ! Another JS rule for some sites
 !+ NOT_OPTIMIZED NOT_PLATFORM(windows, mac, android, ext_ff)
-mineavto.ru,newkaliningrad.ru,eku.ru,kramola.info,farap.ru,vesti.ru,myslo.ru,pravda.ru,vesti.ru,itog.info,myshows.me,neolove.ru#%#AG_abortInlineScript(/\['charCodeAt'\]/, 'atob');
+mineavto.ru,newkaliningrad.ru,eku.ru,kramola.info,farap.ru,vesti.ru,myslo.ru,pravda.ru,vesti.ru,itog.info,myshows.me,neolove.ru#%#//scriptlet("abort-current-inline-script", "atob", "['charCodeAt']")
```

Comment:
As I can see, these sites are russian and this rule should be in AdGuard Russian, but I don't see the need for its application.

Reply:
Better do not make different changes in one PR. We can move them anytime.

Reply:
Ok.